### ISSUE TITLE
The validation set result is 0

### ISSUE THREAD

**Issue by ke-dev** [hustvl/TopFormer#18]
Hi, thanks for your excellent work.
When I used toformer to train the ADE20K dataset, the result of the validation set appeared 0. Have you ever encountered this situation?
![337ba331169f76b1985b94671ea2f86](https://user-images.githubusercontent.com/59656387/172557401-34e329f1-fba0-4377-be3f-eed689380a19.png)
This is the command I use
`bash tools/dist_train.sh local_configs/topformer/topformer_tiny_448x448_160k_2x8_ade20k.py 1`
look forward to you reply.

**Comment by speedinghzl** [comment]
From [issue](https://github.com/hustvl/TopFormer/issues/16), the Imagenet pre-trained model is needed for initialization. You can find the download links from [ReadMe](https://github.com/hustvl/TopFormer#readme).
<img width="898" alt="image" src="https://user-images.githubusercontent.com/4509744/172760823-bc20b4f5-9c9f-4ad7-a58f-3f2144fd4b52.png">
